TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -1017,59 +1023,220 @@ private void WriteVariantResults()
             }
             foreach (var peptide in confidentVariantPeps)
             {
-                if (peptide.FdrInfo.QValue <= 0.01 && peptide.FdrInfo.QValueNotch <= 0.01 && peptide.IsDecoy == false)
+                if (peptide.FdrInfo.QValue <= 0.01 && peptide.FdrInfo.QValueNotch <= 0.01 && !peptide.IsDecoy && !peptide.IsContaminant)
```

Comment:
The first set of cutoffs at line 946 exist to filter the output that is written in the variant specific peptide output file. The cutoffs applied here are utilized for the variant result summary file in which we want to count the number of variants of each type at 1% group FDR which may be different from the CommonParameter settings. We also want to exclude any variant or contaminant peptides from these calculations so that the totals provided are only target peptide identifications.

Reply:
still feel that the cut-offs should be coincident. if user specifies 5% then 5% for all seems right. No issue with filtering decoys/contaminants. Doubtful that any contaminants have variants and so I don't think that's probably a major thing.

Reply:
The default for the qValue output filter is 1.0 meaning that all peptides or psms up to 100% FDR is written into the psm file, not 1% FDR or (0.01). I do not think that we should have this value be set to the default variant result filter for the output text file. I will do some work to try and accommodate, if the value if no the default, to have the results be filtered to the user selected FDR